src/main/java/ai/starlake/transpiler/schema/JdbcColumn.java

```diff
@@ -48,6 +58,7 @@ public class JdbcColumn implements Comparable<JdbcColumn> {
   public String scopeCatalog;
   public String scopeSchema;
   public String scopeTable;
+  public String scopeColumn;
```

Comment:
Why is this needed? Can you provide a simple example for illustration please where the column name is different from scope column name? In my understanding this is what Alias is used for?

Reply:
In case of SQL statements like this:
select * from a, (select cola id, colb b1 from mytable) b where a.id=b.id

the resulting lineage in scope attribute/value references the renamed columns (they are copied into the resultset during analysis) adding scopeColumn allows to preserve the orginal name thus providing the correct (for my needs - the original source of data) scope information.